modules/docker/Makefile.build

```diff
@@ -0,0 +1,12 @@
+.PHONY: docker\:build
+## Use DOCKER_IMAGE_NAME envvar to specify docker image with tags
+## Build docker image
+docker\:build: $(DOCKER)
```

Comment:
When using `\:` in target names, the build deps (in this case `$(DOCKER)`) will not be called. Haven't figured out why.